adding MP3 Pixel tracker

Jason-Bee opened the pull request:
adding a whole new layout which is more organized plus a lot of newly made pixel icons, thanks to reggie for letting me re-use some older ones and abyss for the huge help!

Jason-Bee commented:
![Unbenannt](https://github.com/randovania/randovania/assets/132858440/ea9c508c-ef0d-4f2d-bd1a-fed54fa18963)
here a picture of the finished product

Miepee commented:
I'll leave it up to the Corruption Maintainers to decide, but a few comments on my side (which I wouldn't be blocking the PR for)
- I don't mean to downplay your work here, but some of the sprites really give a feeling of "took sprites and threw them together", due to how inconsistent they look. Like Morph Ball having different shading than hyper ball, which all look completely different to the spider ball, command visor sticking out like a sore thumb, or scan visor having no shading at all compared to xray.
- The grapple upgrades look really off to me right now. Currently they give me more a feeling of "trident" than "grapple", which I assume is due to missing shading, the neck being a bit too long, and the base being invisible in non-collected mode

Jason-Bee commented:
" Like Morph Ball having different shading than hyper ball, which all look completely different to the spider ball,"
 due to colour difference, fixed the spider issue tho to keep some consistency.
 
 command visor sticking out like a sore thumb, or scan visor having no shading at all compared to xray.
 Scan shading is adjusted, command and Xray cant be changed. xray already has a lot of shading but white on more white is hard to read, changing command would take away the readability of command compared to Combat.
 
 The grapple upgrades look really off to me right now. Currently they give me more a feeling of "trident" than "grapple"
yea i noticed that aswell after you pointed it out, updated the handle to be more fitting, not perfect. but for sure better then before @Miepee

Jason-Bee commented:
![image](https://github.com/randovania/randovania/assets/132858440/f3dce521-e63a-498f-9428-e2791d128dd2)

ThanatosGit commented:
Uh, anyone feels responsible for this?

henriquegemignani commented:
I kinda don't want more themes for any game before having a decent theme selector, but corruption has just one theme so far

Miepee commented:
Abyss was working on redrawing some sprites, so there will probably be more commits pushed to this

Miepee commented:
I'll mark this as draft until then.

Miepee commented on randovania/data/gui_assets/tracker/pixel-icons/mp3/Command_visor.png:
Please have a consistent naming scheme for your files.
I don't care if you decide on lower_case_snake, Upper_Case_Snake, PascalCase or camelCase, as long as your files are consistent with it.

Miepee replied:
This has not been fixed. The files still have an inconsistent naming scheme.

Miepee commented:
@Jason-Bee Bump

Miepee commented on .vscode/settings.json:
please remove this file

Miepee commented on randovania/data/gui_assets/tracker/prime3-Pixel.json:
please rename this file to `prime3-pixel.json`  or it will fail loading on linux and macos.

Miepee reviewed (approved):
no oneelse feels responsible, and looks fine to me, so i'll merge
